src/ControllerStatusIO.h

```diff
@@ -142,6 +142,8 @@ extern int Ros_Controller_GetAlarmCode();
 //retrieve all active alarms (and a possible error) and store them in 'active_alarms'
 extern int Ros_Controller_GetActiveAlarmCodes(USHORT active_alarms[MAX_ALARM_COUNT + MAX_ERROR_COUNT]);
 
+//TODO(gavanderhoorn): make static, see comment on definition
+extern BOOL Ros_Controller_ShouldWarnNoCalibDataLoaded(Controller const* controller, BOOL bCalibLoadedOk, BOOL bPublishTfEnabled);
```

Comment:
I don't get why this is in the 'public' header

Reply:
See my earlier comment.